xontrib/jedi.xsh

```diff
@@ -31,10 +31,13 @@ def complete_jedi(prefix, line, start, end, ctx):
     src = builtins.__xonsh__.shell.shell.accumulated_inputs + line
     script = jedi.api.Interpreter(src, [ctx], column=end)
     if builtins.__xonsh__.env.get('CASE_SENSITIVE_COMPLETIONS'):
-        rtn = {x.name_with_symbols for x in script.completions()
-               if x.name_with_symbols.startswith(prefix)}
+       rtn = {x.name_with_symbols for x in script.completions()
```

Comment:
`script.completions()` could also throw an exception here, fyi. I was thinking of either wrapping the whole if / else or returning early.

Reply:
Yeah, it could in theory, but I didn't know if `completions()` returned a list or an iterator